### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by VolkerLaude** [joBr99/nspanel-lovelace-ui#1072]
### PROBLEM DESCRIPTION
Waiting for content - This is taking longer than usual on the screen
Panel doesn't load configuration

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Provide your apps.yaml (or your config):
```
  apps.yaml here:
---
nspanel-1:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/Wohnzimmerlampe/RESULT"
    panelSendTopic: "cmnd/Wohnzimmerlampe/CustomSend"
    model: eu
    sleepTimeout: 20
    sleepBrightness:
      - time: "7:00:00"
        value: 10
      - time: "23:00:00"
        value: 0
    locale: "de_DE"
    screensaver:
      entity: weather.k3ll3r
    cards:
      - type: cardEntities
        entities:
          - entity: switch.example_item
          - entity: light.example_item
        title: Example 1
      - type: cardGrid
        entities:
          - entity: switch.example_item
          - entity: light.example_item
        title: Example 2


```
- [ ] Go to Settings>Add-ons>AppDaemon>Log (or the output of your TS script) and then, provide the output of the log during your issue/bug occurs:
```
  Log output here:
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service base-addon-banner: starting
-----------------------------------------------------------

 Add-on: AppDaemon
 Python Apps and Dashboard using AppDaemon 4.x for Home Assistant
-----------------------------------------------------------
 Add-on version: 0.15.2
 You are running the latest version of this add-on.
 System: Home Assistant OS 11.1  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.11.3
 Home Assistant Supervisor: 2023.11.3
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-appdaemon: starting
Looking in indexes: https://pypi.org/simple, https://wheels.home-assistant.io/musllinux-index/
Collecting Babel
  Downloading Babel-2.13.1-py3-none-any.whl (10.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 17.9 MB/s eta 0:00:00
Installing collected packages: Babel
Successfully installed Babel-2.13.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
s6-rc: info: service init-appdaemon successfully started
s6-rc: info: service appdaemon: starting
s6-rc: info: service appdaemon successfully started
s6-rc: info: service legacy-services: starting
[20:53:08] INFO: Starting AppDaemon...
s6-rc: info: service legacy-services successfully started
2023-11-27 20:53:16.580741 INFO AppDaemon: AppDaemon Version 4.4.2 starting
2023-11-27 20:53:16.581270 INFO AppDaemon: Python version is 3.11.6
2023-11-27 20:53:16.581751 INFO AppDaemon: Configuration read from: /config/appdaemon.yaml
2023-11-27 20:53:16.582244 INFO AppDaemon: Added log: AppDaemon
2023-11-27 20:53:16.582792 INFO AppDaemon: Added log: Error
2023-11-27 20:53:16.583366 INFO AppDaemon: Added log: Access
2023-11-27 20:53:16.583819 INFO AppDaemon: Added log: Diag
2023-11-27 20:53:17.084246 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
2023-11-27 20:53:17.563539 INFO HASS: HASS Plugin Initializing
2023-11-27 20:53:17.564100 WARNING HASS: ha_url not found in HASS configuration - module not initialized
2023-11-27 20:53:17.564606 INFO HASS: HASS Plugin initialization complete
2023-11-27 20:53:17.565248 INFO AppDaemon: Loading Plugin MQTT using class MqttPlugin from module mqttplugin
2023-11-27 20:53:17.737032 INFO MQTT: MQTT Plugin Initializing
2023-11-27 20:53:17.737642 INFO MQTT: Using 'appdaemon/status' as Will Topic
2023-11-27 20:53:17.738126 INFO MQTT: Using 'appdaemon/status' as Birth Topic
2023-11-27 20:53:17.739634 INFO AppDaemon: HTTP is disabled
2023-11-27 20:53:17.764703 INFO HASS: Connected to Home Assistant 2023.11.3
2023-11-27 20:53:17.792887 INFO AppDaemon: Starting Apps with 0 workers and 0 pins
2023-11-27 20:53:17.933294 INFO HASS: Evaluating startup conditions
2023-11-27 20:53:17.966444 INFO HASS: Startup condition met: hass state=RUNNING
2023-11-27 20:53:17.967227 INFO HASS: All startup conditions met
2023-11-27 20:53:18.085404 INFO AppDaemon: Got initial state from namespace default
2023-11-27 20:53:18.355458 INFO MQTT: Connected to Broker at URL core-mosquitto.local.hass.io:1883
2023-11-27 20:53:18.798803 INFO AppDaemon: Got initial state from namespace mqtt
2023-11-27 20:53:18.799658 INFO MQTT: MQTT Plugin initialization complete
2023-11-27 20:53:20.803514 INFO AppDaemon: Scheduler running in realtime
2023-11-27 20:53:20.810078 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/ to module import path
2023-11-27 20:53:20.810947 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui to module import path
2023-11-27 20:53:20.811934 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend to module import path
2023-11-27 20:53:20.813022 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations to module import path
2023-11-27 20:53:20.813980 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/backend to module import path
2023-11-27 20:53:20.814833 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/frontend to module import path
2023-11-27 20:53:20.840786 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/nspanel-lovelace-ui.py - ignoring
2023-11-27 20:53:20.842486 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/helper.py - ignoring
2023-11-27 20:53:20.843914 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/icon_mapping.py - ignoring
2023-11-27 20:53:20.845149 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py - ignoring
2023-11-27 20:53:20.846457 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/controller.py - ignoring
2023-11-27 20:53:20.847765 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/config.py - ignoring
2023-11-27 20:53:20.849085 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/apis.py - ignoring
2023-11-27 20:53:20.850262 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/updater.py - ignoring
2023-11-27 20:53:20.851611 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/icons.py - ignoring
2023-11-27 20:53:20.852881 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/theme.py - ignoring
2023-11-27 20:53:20.854217 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/localization.py - ignoring
2023-11-27 20:53:20.855631 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/mqtt.py - ignoring
2023-11-27 20:53:20.857505 INFO AppDaemon: App initialization complete

```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Console Tasmota
restart 1

Reload AppDaemon

### EXPECTED BEHAVIOUR
Panel loads configuration AppDaemon

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
_Please note here in case you are using ioBroker_

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_
13.2.0(nspanel) US-P

**Comment by joBr99** [comment]
please move it into the apps folder

**Comment by VolkerLaude** [comment]
![image](https://github.com/joBr99/nspanel-lovelace-ui/assets/54547517/3871ebd0-80ab-42bf-9da8-c25490f3828c)

It is bit different now:
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service base-addon-banner: starting

-----------------------------------------------------------
 Add-on: AppDaemon
 Python Apps and Dashboard using AppDaemon 4.x for Home Assistant
-----------------------------------------------------------
 Add-on version: 0.15.2
 You are running the latest version of this add-on.
 System: Home Assistant OS 11.1  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.11.3
 Home Assistant Supervisor: 2023.11.3
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-appdaemon: starting
Looking in indexes: https://pypi.org/simple, https://wheels.home-assistant.io/musllinux-index/
Collecting Babel
  Downloading Babel-2.13.1-py3-none-any.whl (10.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 15.8 MB/s eta 0:00:00
Installing collected packages: Babel
Successfully installed Babel-2.13.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
s6-rc: info: service init-appdaemon successfully started
s6-rc: info: service appdaemon: starting
s6-rc: info: service appdaemon successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
[21:31:52] INFO: Starting AppDaemon...
2023-11-27 21:32:00.463652 INFO AppDaemon: AppDaemon Version 4.4.2 starting
2023-11-27 21:32:00.464182 INFO AppDaemon: Python version is 3.11.6
2023-11-27 21:32:00.466118 INFO AppDaemon: Configuration read from: /config/appdaemon.yaml
2023-11-27 21:32:00.466586 INFO AppDaemon: Added log: AppDaemon
2023-11-27 21:32:00.468421 INFO AppDaemon: Added log: Error
2023-11-27 21:32:00.468869 INFO AppDaemon: Added log: Access
2023-11-27 21:32:00.469136 INFO AppDaemon: Added log: Diag
2023-11-27 21:32:00.985533 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
2023-11-27 21:32:01.463695 INFO HASS: HASS Plugin Initializing
2023-11-27 21:32:01.464036 WARNING HASS: ha_url not found in HASS configuration - module not initialized
2023-11-27 21:32:01.464520 INFO HASS: HASS Plugin initialization complete
2023-11-27 21:32:01.465408 INFO AppDaemon: Loading Plugin MQTT using class MqttPlugin from module mqttplugin
2023-11-27 21:32:01.640795 INFO MQTT: MQTT Plugin Initializing
2023-11-27 21:32:01.641175 INFO MQTT: Using 'appdaemon/status' as Will Topic
2023-11-27 21:32:01.641677 INFO MQTT: Using 'appdaemon/status' as Birth Topic
2023-11-27 21:32:01.642892 INFO AppDaemon: HTTP is disabled
2023-11-27 21:32:01.665597 INFO HASS: Connected to Home Assistant 2023.11.3
2023-11-27 21:32:01.721741 INFO AppDaemon: App 'nspanel-1' added
2023-11-27 21:32:01.724070 INFO AppDaemon: Found 1 active apps
2023-11-27 21:32:01.725600 INFO AppDaemon: Found 0 inactive apps
2023-11-27 21:32:01.726379 INFO AppDaemon: Found 0 global libraries
2023-11-27 21:32:01.726872 INFO AppDaemon: Starting Apps with 1 workers and 1 pins
2023-11-27 21:32:01.826166 INFO HASS: Evaluating startup conditions
2023-11-27 21:32:01.844462 INFO HASS: Startup condition met: hass state=RUNNING
2023-11-27 21:32:01.845478 INFO HASS: All startup conditions met
2023-11-27 21:32:01.965225 INFO AppDaemon: Got initial state from namespace default
2023-11-27 21:32:02.238022 INFO MQTT: Connected to Broker at URL core-mosquitto.local.hass.io:1883
2023-11-27 21:32:02.735064 INFO AppDaemon: Got initial state from namespace mqtt
2023-11-27 21:32:02.737131 INFO MQTT: MQTT Plugin initialization complete
2023-11-27 21:32:04.739080 INFO AppDaemon: Scheduler running in realtime
2023-11-27 21:32:04.744196 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/ to module import path
2023-11-27 21:32:04.745568 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui to module import path
2023-11-27 21:32:04.747049 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend to module import path
2023-11-27 21:32:04.748271 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations to module import path
2023-11-27 21:32:04.749492 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/backend to module import path
2023-11-27 21:32:04.750775 INFO AppDaemon: Adding /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/frontend to module import path
2023-11-27 21:32:04.768059 INFO AppDaemon: Loading App Module: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/nspanel-lovelace-ui.py
2023-11-27 21:32:05.208651 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/helper.py - ignoring
2023-11-27 21:32:05.209772 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/icon_mapping.py - ignoring
2023-11-27 21:32:05.211289 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py - ignoring
2023-11-27 21:32:05.212723 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/controller.py - ignoring
2023-11-27 21:32:05.214392 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/config.py - ignoring
2023-11-27 21:32:05.215902 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/apis.py - ignoring
2023-11-27 21:32:05.217591 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/updater.py - ignoring
2023-11-27 21:32:05.219475 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/icons.py - ignoring
2023-11-27 21:32:05.221212 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/theme.py - ignoring
2023-11-27 21:32:05.223651 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/localization.py - ignoring
2023-11-27 21:32:05.227245 WARNING AppDaemon: No app description found for: /homeassistant/appdaemon/apps/nspanel-lovelace-ui/luibackend/mqtt.py - ignoring
2023-11-27 21:32:05.236382 INFO AppDaemon: Loading app nspanel-1 using class NsPanelLovelaceUIManager from module nspanel-lovelace-ui
2023-11-27 21:32:05.240283 INFO AppDaemon: Calling initialize() for nspanel-1
2023-11-27 21:32:05.383216 INFO nspanel-1: Starting
2023-11-27 21:32:05.394082 INFO nspanel-1: Input config: {'panelRecvTopic': 'tele/Wohnzimmer/RESULT', 'panelSendTopic': 'cmnd/Wohnzimmer/CustomSend', 'model': 'eu', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 0}], 'locale': 'de_DE', 'screensaver': {'entity': 'weather.k3ll3r'}, 'cards': [{'type': 'cardEntities', 'entities': [{'entity': 'switch.example_item'}, {'entity': 'light.example_item'}], 'title': 'Example 1'}, {'type': 'cardGrid', 'entities': [{'entity': 'switch.example_item'}, {'entity': 'light.example_item'}], 'title': 'Example 2'}]}
2023-11-27 21:32:05.397057 INFO nspanel-1: Loaded config: {'panelRecvTopic': 'tele/Wohnzimmer/RESULT', 'panelSendTopic': 'cmnd/Wohnzimmer/CustomSend', 'updateMode': 'auto-notify', 'model': 'eu', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 0}], 'screenBrightness': 100, 'defaultBackgroundColor': 'ha-dark', 'featureExperimentalSliders': False, 'sleepTracking': None, 'sleepTrackingZones': ['not_home', 'off'], 'sleepOverride': None, 'locale': 'de_DE', 'timeFormat': '%H:%M', 'dateFormatBabel': 'full', 'dateAdditionalTemplate': '', 'timeAdditionalTemplate': '', 'dateFormat': '%A, %d. %B %Y', 'cards': [{'type': 'cardEntities', 'entities': [{'entity': 'switch.example_item'}, {'entity': 'light.example_item'}], 'title': 'Example 1'}, {'type': 'cardGrid', 'entities': [{'entity': 'switch.example_item'}, {'entity': 'light.example_item'}], 'title': 'Example 2'}], 'screensaver': {'type': 'screensaver', 'entity': 'weather.k3ll3r', 'weatherUnit': 'celsius', 'forecastSkip': 0, 'weatherOverrideForecast1': None, 'weatherOverrideForecast2': None, 'weatherOverrideForecast3': None, 'weatherOverrideForecast4': None, 'doubleTapToUnlock': False, 'alternativeLayout': False, 'defaultCard': None, 'key': 'screensaver'}, 'hiddenCards': []}
2023-11-27 21:32:05.407558 INFO nspanel-1: Sending MQTT Message: pageType~pageStartup
2023-11-27 21:32:05.433108 INFO nspanel-1: gtest123: ['switch.example_item', 'light.example_item', 'switch.example_item', 'light.example_item', 'weather.k3ll3r']
2023-11-27 21:32:05.436401 INFO nspanel-1: Registering callbacks for the following items: ['switch.example_item', 'light.example_item', 'switch.example_item', 'light.example_item', 'weather.k3ll3r']
2023-11-27 21:32:05.447532 INFO nspanel-1: Started (v4.3.1)
2023-11-27 21:32:05.450265 INFO AppDaemon: App initialization complete

**Comment by VolkerLaude** [comment]
But the panel still load the config...

**Comment by VolkerLaude** [comment]
Here it is...
![image](https://github.com/joBr99/nspanel-lovelace-ui/assets/54547517/9e4add9d-097d-4403-95f2-420699a26d67)

**Comment by joBr99** [comment]
Your topics don't match - you've used `Wohnzimmer` in your apps.yaml and `Wohnzimmerlampe` in tasmota.

**Comment by VolkerLaude** [comment]
You made my day. Silly me. Maybe I was staring at my screen for too long.
But where did I miss the part in the procedure with the path /config/appdaemon/apps/apps.yaml?
👍

**Comment by joBr99** [comment]
https://docs.nspanel.pky.eu/configure_mqtt/#configure-nspanel-on-appdaemon

The path is there; however there are no explicit instructions to move the file automatically created by the addon.

Last Addon Upgrade messed everything up ... before version 15 everything was just in /config/appdaemon ... with the last update everything moved to the new folder. However HACS will only write the old location and also doesn't have access to the new location. So with this line in appdaemon.yaml we are configuring it back to the old location to fix HACS, which also requires to move the apps.yaml.
TLDR, it's a mess ... but I'm working on an addon without appdaemon.
